taworks/taform/views.py

```diff
@@ -47,22 +54,31 @@ def application_submitted(request):
     return render(request, 'taform/application_submitted.html')
 
 def course_list(request):
```

Comment:
Do we need @login_required here?

https://docs.djangoproject.com/en/1.11/topics/auth/default/